### PR TITLE
Updated to 6.18.0 rooster which includes the improvement on undo plugin marker and fixes the last word edit bug.

### PR DESCRIPTION
**Pull request by adriantran**
Also fix some minor bugs.

### CODE REVIEW

File: packages/roosterjs-react-editor/lib/components/LeanRooster.tsx [hunk 1]
```diff
@@ -113,6 +111,36 @@ export default class LeanRooster extends React.Component<LeanRoosterProps, {}> {
...
+    public componentWillReceiveProps(nextProps: LeanRoosterProps, nextState: {}) {
+        const div = this._contentDiv;
+        if (!div) {
+            return;
+        }
+
+        const { className, readonly, isRtl, placeholder } = nextProps;
+        
+        if (className !== this.props.className) {
+            div.setAttribute("class", this._getClassName(nextProps));
```
Comment (pranavgk) [inline packages/roosterjs-react-editor/lib/components/LeanRooster.tsx]: This will remove any other existing class names on this div. Is that acceptable?

Comment (pranavgk) [reply packages/roosterjs-react-editor/lib/components/LeanRooster.tsx]: Never mind, noticed you call _getClassName. Ignore. :)

File: sample/script/FocusOutShellSample.tsx [hunk 2]
```diff
@@ -192,7 +199,14 @@ function createEditor(name: string, loadEmojiStrings: boolean = false): JSX.Elem
                         calloutOnDismiss={calloutOnDismiss}
                         imageManager={imageManager}
                         ref={commandBarOnRef}
-                        onButtonClicked={buttonKey => console.log(buttonKey)}
+                        onButtonClicked={buttonKey => {
+                            if (buttonKey === ButtonKeys.InsertImage) {
+                                suppressBlur = true;
+                            }
+
+                            console.log(buttonKey)
```
Comment (pranavgk) [inline sample/script/FocusOutShellSample.tsx]: Delete

Comment (adriantran) [reply sample/script/FocusOutShellSample.tsx]: it's intentional

**Pull request merged**